MSAL/src/native_auth/network/errors/reset_password/MSALNativeAuthResetPasswordChallengeResponseError.swift

```diff
@@ -52,6 +52,8 @@ extension MSALNativeAuthResetPasswordChallengeResponseError {
              .unsupportedChallengeType,
              .expiredToken:
             return .init(type: .generalError, message: errorDescription)
+        case .none:
```

Comment:
This is for the case when error is `nil`

Reply:
just for personal preference I'd prefer to have a guard let error at the begging of the method. But if you like more in this way it's ok